File: frontend/src/app/domain/report/dto/performans-ozet-dto.ts
import {CiroDto} from "./ciro-dto";
import {TahsilatDto} from "./tahsilat-dto";
import {SiparisDto} from "./siparis-dto";
import {HedefDto} from "./hedef-dto";

export class PerformansOzetDto {

  donem: string;
  aylikCiro: CiroDto;
  yillikCiro: CiroDto;
  aylikHedef: HedefDto;
  yillikHedef: HedefDto;
  aylikTahsilat: TahsilatDto;
  yillikTahsilat: TahsilatDto;
  aylikSiparis: SiparisDto;
  yillikSiparis: SiparisDto;

}

File: backend/src/main/java/com/aymer/sirketimceptebackend/report/dto/PerformansOzetDto.java
package com.aymer.sirketimceptebackend.report.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@Builder
public class PerformansOzetDto {

    private Donem donem;
    private CiroDto aylikCiro;
    private CiroDto yillikCiro;
    private HedefDto aylikHedef;
    private HedefDto yillikHedef;
    private TahsilatDto aylikTahsilat;
    private TahsilatDto yillikTahsilat;
    private SiparisDto aylikSiparis;
    private SiparisDto yillikSiparis;

}
